requirements_windows.txt

```diff
@@ -1,82 +1,86 @@
-anaconda-client==1.7.2
-anaconda-navigator==1.9.12
-attrs==19.3.0
-backcall==0.1.0
-beautifulsoup4==4.8.2
-bleach==3.1.2
-certifi==2019.11.28
-cffi==1.14.0
-chardet==3.0.4
-clyent==1.2.2
-colorama==0.4.3
-conda==4.8.2
-conda-build==3.18.12
-conda-package-handling==1.6.0
-cryptography==2.8
-decorator==4.4.2
-defusedxml==0.6.0
-entrypoints==0.3
-filelock==3.0.10
-glob2==0.7
-idna==2.9
-importlib-metadata==1.5.0
-ipykernel==5.1.4
-ipython==7.13.0
-ipython-genutils==0.2.0
-jedi==0.16.0
-Jinja2==2.11.1
-jsonschema==3.2.0
-jupyter-client==6.0.0
-jupyter-core==4.6.3
-libarchive-c==2.9
-MarkupSafe==1.1.1
-menuinst==1.4.16
-mistune==0.8.4
-nbconvert==5.6.1
-nbformat==5.0.4
-notebook==6.0.1
-nltk==3.4.5
-olefile==0.46
-pandocfilters==1.4.2
-parso==0.6.2
-pickleshare==0.7.5
-Pillow==7.0.0
-pkginfo==1.5.0.1
-prometheus-client==0.7.1
-prompt-toolkit==3.0.3
-psutil==5.7.0
-pycosat==0.6.3
-pycparser==2.20
-Pygments==2.5.2
-pyOpenSSL==19.1.0
-pyrsistent==0.14.11
-PySocks==1.7.1
-python-dateutil==2.8.1
-pytz==2019.3
-pywin32==227
-pypiwin32==0.5.7
-pywinpty==0.5.7
-PyYAML==5.3
-pyzmq==19.0.0
-QtPy==1.9.0
-requests==2.23.0
-ruamel-yaml==0.15.80
-Send2Trash==1.5.0
-SpeechRecognition==3.8.1
-sip==4.19.20
-six==1.14.0
-soupsieve==1.9.4
-terminado==0.8.3
-testpath==0.4.4
-tornado==6.0.4
-tqdm==4.43.0
-traitlets==4.3.3
-urllib3==1.25.7
-wcwidth==0.1.8
-webencodings==0.5.1
-win-inet-pton==1.1.0
-wincertstore==0.2
-xmltodict==0.12.0
-zipp==3.1.0
-pyjokes==0.6.0
-(base)
+anaconda-client
+#anaconda-navigator
```

Comment:
Why this is commented?

Reply:
Actually, I don't use anaconda. So, Installing anaconda-navigator is giving error and some of other installation related to anaconda are giving error and i have commented them. 
Should I remove the comments or resolve it or leave as it is?

Reply:
remove them for now, you can add that in separate pr to resolve the issue

Reply:
Done! Please check it.